src/questions/questions.js

```diff
@@ -212,10 +225,7 @@ export const NETWORK_CONFIRM = {
 }
 
 export const GENERATE_NAME = {
-  type: 'input',
-  name: 'name',
-  message: 'What would you like to call this network?',
-  validate: (input) => input.trim() !== '' || 'Network name must not be blank.',
+  ...NETWORK_NAME,
```

Comment:
generate_name doesn't have answers.deployment, so the check doesn't get caught in regenerating a network with too long of a name

Reply:
good catch, now it loads the config and grabs the deployment similar to how it grabs the existing name